roles/validations/tasks/xmlcreate.yml

```diff
@@ -0,0 +1,58 @@
+---
+- name: Get validation start time from system
```

Comment:
Why not use the `now(fmt='%s.%f')`? It's the Ansible way of doing this.
A simple cast to `float` will convert it to a numeric timestamp

Reply:
Updated to use: now(fmt='%s.%f')